dirsrvtests/tests/suites/subentries/subentries_test.py

```diff
@@ -0,0 +1,201 @@
+# --- BEGIN COPYRIGHT BLOCK ---
+# Copyright (C) 2021 RED Hat, Inc.
```

Comment:
nit: capitalization

Reply:
That should teach me not to copy paste existing copyright notices ;)